test/components/preprocessors/test_document_cleaner.py

```diff
@@ -211,7 +211,6 @@ def test_other_document_fields_are_not_lost(self):
         cleaner = DocumentCleaner(keep_id=True)
         document = Document(
             content="This is a text with some words. \nThere is a second sentence. \nAnd there is a third sentence.\n",
-            dataframe=DataFrame({"col1": [1], "col2": [2]}),
```

Comment:
Let's also remove `from pandas import DataFrame` from this file.

Reply:
https://github.com/deepset-ai/haystack/pull/8906/commits/bf5b84cbe3a1102bfd15c5dfa1aeea61f7660741